.idea/modules.xml

```diff
@@ -2,7 +2,7 @@
 <project version="4">
```

Comment:
this file shouldn't be uploaded

Reply:
Done

Reply:
/cc @fathallah92  I've just add this file to .ignore file.